storage/store.go

```diff
@@ -1151,6 +1153,74 @@ func (s *Store) processRaft() {
 					log.Fatal(err)
 				}
 
+			case *multiraft.EventLeaderElection:
+				if e.NodeID == 0 {
+					// Election in progress.
+					continue
+				}
+				// Only the store housing the election winner gets to
+				// proceed.
+				groupID = int64(e.GroupID)
+				_, err := s.GetRange(groupID)
+				if err != nil {
+					log.Warning(err)
+					continue
+				}
+				if e.NodeID != s.RaftNodeID() {
+					// TODO(tschottdorf): Can and should still immediately revoke any
```

Comment:
There can be no successful elections while a node has the lease, so we should log an error (or maybe panic, at least for now) if this happens while we have the lease.